core/templates/dev/head/domain/state/StateObjectFactory.ts

```diff
@@ -58,6 +58,11 @@ export class State {
   }
   setName(newName: string): void {
     this.name = newName;
+    this.updateInteractionDestName(newName);
```

Comment:
Why do this?

Reply:
I've removed this function. I thought the `name` property was related to the `interaction` property before, but I checked it further and it was my mistake.